src/runtime/HalideBuffer.h

```diff
@@ -1205,29 +1205,39 @@ class Buffer {
     }
 
     /** Set the min coordinate of an image in the first N dimensions */
-    template<typename ...Args>
-    void set_min(Args... args) {
-        assert(sizeof...(args) <= (size_t)dimensions());
+    // @{
+    void set_min(std::vector<int> mins) {
```

Comment:
Why not:
 
    const std::vector<int> &

Reply:
Done